### ISSUE TITLE
ranger 1.9: all autofs mounts triggered on start?

### ISSUE THREAD

**Issue by eMPee584** [ranger/ranger#1365]
Since the switch to 1.9 all autofs mounts seem to be triggered when starting ranger, as well as every time the bookmark menu is requested. As not all mounts are available, and even though timeouts are defined, ranger stalls indefinitely. Disabling autofs allows fluid use as before.

**Comment by toonn** [comment]
And this worked on earlier versions?

**Comment by eMPee584** [comment]
yes, just tested it. ranger_1.8.1-0.2 behaves very fine, while 1.9.2 stalls..
ah yeah I see stat() calls to all bookmarked paths on directory change…

**Comment by j-clem** [comment]
Same issue with systemd automount as well.

**Comment by eMPee584** [comment]
Ha!
```
Bisecting: 388 revisions left to test after this (roughly 9 steps)
…
924135e1934a01faef64e94d2425d23f9790b6cf is the first bad commit
commit 924135e1934a01faef64e94d2425d23f9790b6cf
Author: nfnty <git@nfnty.se>
Date:   Sun Jan 29 19:58:58 2017 +0100

    core.main: Check that paths are accessible before starting

    Remove deprecated use as file launcher

    Fixes #560
    Fixes #558
    Fixes #424

:040000 040000 19f07782ced476c7af078ee520cdfdbc381a4163 6d9d9e99a08f76318c06109209e0e0acfd6e4cc8 M      ranger                                                

```

**Comment by eMPee584** [comment]
mmh.. tried to gdb into ranger to see which paths are checked at exactly which time (hanging also occurs when invoking the bookmarks menu, not only on start) but couldn't quite get that to work.
